<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>363</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Opinsys Installer Script</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_asenna">
       <attribute name="title">
        <string>Asenna</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="Ohjelma">
          <property name="minimumSize">
           <size>
            <width>407</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Valitse asennettava ohjelma</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_ohjelmat_asennus">
          <property name="minimumSize">
           <size>
            <width>407</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Steam</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Minecraft Launcher</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Discord</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vesktop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Brave</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paketin hallinta</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tilanne">
          <property name="text">
           <string>Tilanne: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_poista">
            <property name="text">
             <string>Poista</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_asenna">
            <property name="text">
             <string>Asenna</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_kaynnista">
       <attribute name="title">
        <string>Käynnistä</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Voit hakea ohjelmia painamalla ensin Windows näppäintä ja sen jälkeen kirjoittamalla ohjelman nimen.</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
